integration/test_collection.py

```diff
@@ -1500,3 +1500,15 @@ def test_return_properties_with_type_hint_generic(
     objects = collection.query.fetch_objects().objects
     assert len(objects) == 1
     assert objects[0].properties["name"] == value
+
+
+def test_return_blob_property(collection_factory: CollectionFactory) -> None:
+    collection = collection_factory(
+        properties=[
+            Property(name="blob", data_type=DataType.BLOB),
+        ]
+    )
+    collection.data.insert(properties={"blob": WEAVIATE_LOGO_OLD_ENCODED})
```

Comment:
could you add another object (can be the same content) with batch insert?

Reply:
I meant to have 
```
    uuid1 = collection.data.insert(properties={"blob": WEAVIATE_LOGO_OLD_ENCODED})
    uuid2 = collection.data.insert_many([{"blob": WEAVIATE_LOGO_OLD_ENCODED}])
```

so we test REST and GRPC (and then fetching both and checking them)